packages/typespec-go/src/tcgcadapter/types.ts

```diff
@@ -45,7 +45,7 @@ export class typeAdapter {
     const ifaceTypes = new Array<InterfaceTypeSdkModelType>();
     for (const modelType of sdkContext.sdkPackage.models) {
       if (modelType.name.length === 0) {
-        // tcgc creates some unamed models for spread params.
+        // tcgc creates some unnamed models for spread params.
```

Comment:
I wonder if this is still true? Don't have to change it now, just curious if you know.

Reply:
tcgc should not give any models without name. but it seems there are some anonymous cases that will break this. it's safe to keep it for now.